WikipediaUnitTests/Code/TWNStringsTests.m

```diff
@@ -118,6 +118,15 @@ + (NSRegularExpression *)reverseTWNTokenRegex {
     return reverseTWNTokenRegex;
 }
 
++ (NSRegularExpression *)reverseTWNTokenRegex {
+    static dispatch_once_t onceToken;
+    static NSRegularExpression *iosTokenRegex;
```

Comment:
is it intentional for `NSRegularExpression` to be `iosTokenRegex` and not `twnTokenRegex` here? same question for `reverseiOSTokenRegex` but `twnTokenRegex ` instead of `iosTokenRegex `

Reply:
nope, fixed